flows/aws/interactive-workflow/README.md

```diff
@@ -20,6 +20,7 @@ Interactive Workflow Demo is designed to guide you through the process of settin
 Before starting, ensure you have the following installed:
 - Python 3.6+
```

Comment:
This should probably be 3.8 since that's the lowest version I think we support now

Reply:
We should probably assume that the user here knows that this requires Python. Do we even need a pre-requisites section?

Reply:
That's probably fair, I think in general we should assume relevant information for these demos outside of perfect specific functionality is familiar to whoever might look at it